package.json

```diff
@@ -37,4 +37,4 @@
         "test": "mocha --reporter spec test/*"
     },
     "version": "2.39.2"
-}
+}
```

Comment:
Clearly unimportant, but most files have a trailing newline. No real reason to change it.